script.js

```diff
@@ -15,4 +15,137 @@ window.onload = () => {
   menu.addEventListener('click', () => {
     popUp.classList.add('hidden');
   });
-};
\ No newline at end of file
+
+  const projectsData = [
+    {
+      title: 'Tonic',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
```

Comment:
- [ ] great job so far. please ensure that the mobile version of popup is full heigth like that of figma

yours
![Screenshot (429)_LI](https://user-images.githubusercontent.com/65298048/188755005-864d643d-6ad8-4bff-a4d4-1b5da77f6451.jpg)
figma
![Screenshot (430)](https://user-images.githubusercontent.com/65298048/188755024-ce2328d8-4aa5-4918-9fe6-050e8702cee1.png)